pppd/ipcp.c

```diff
@@ -1829,7 +1845,12 @@ ipcp_up(fsm *f)
 
 	    /* assign a default route through the interface if required */
 	    if (ipcp_wantoptions[f->unit].default_route) 
+#ifndef __linux__
```

Comment:
Similarly this ifndef and the ones below could go away...

Reply:
yep, done so.